tests/aggregation/tests.py

```diff
@@ -1402,3 +1420,199 @@ def test_coalesced_empty_result_set(self):
                 )['latest_opening'],
                 datetime.datetime,
             )
+
+    def test_aggregation_default_unsupported_by_count(self):
+        msg = 'Count does not allow default.'
+        with self.assertRaisesMessage(TypeError, msg):
+            Count('age', default=0)
+
+    def test_aggregation_default_unset(self):
+        for Aggregate in [Avg, Max, Min, StdDev, Sum, Variance]:
+            with self.subTest(Aggregate):
+                result = (
+                    Author.objects.filter(age__gt=100)
+                    .aggregate(value=Aggregate('age'))
+                )
```

Comment:
The same indentation in other tests:
```suggestion
                result = Author.objects.filter(age__gt=100).aggregate(
                    value=Aggregate('age'),
                )
```